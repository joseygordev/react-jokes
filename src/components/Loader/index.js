import React from 'react';
import { BallBeat } from 'react-pure-loaders';

import theme from '../../config/theme';

const Loader = () => {
  return (
    <BallBeat color={theme.colors.primary} loading />
  );
};

export default Loader;
