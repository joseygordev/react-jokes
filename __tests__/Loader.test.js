import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../src/components/Loader';

describe('Loader', () => {
  it('Should return the Loader component', () => {
    const tree = renderer
      .create(<Loader theme={{ primary: '#000' }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
