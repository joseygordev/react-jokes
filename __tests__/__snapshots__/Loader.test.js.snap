// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Should return the Loader component 1`] = `
<div
  className="css-mh7cz-BallBeat"
>
  <div />
  <div />
  <div />
</div>
`;
